bpf/lib/egress_policies.h

```diff
@@ -38,11 +38,12 @@ bool egress_gw_request_needs_redirect(struct iphdr *ip4, __u32 *tunnel_endpoint)
 	if (!egress_gw_policy)
 		return false;
 
-	/* If the gateway IP is 0.0.0.0 it means this is an excluded CIDR, so
+	/* If the gateway IP is 0.0.0.0 or 0.0.0.1 it means this is an excluded CIDR, so
 	 * skip redirection
 	 */
-	if (!egress_gw_policy->gateway_ip)
+	if (!egress_gw_policy->gateway_ip || egress_gw_policy->gateway_ip == 1)
```

Comment:
I think at this point we should define a couple of constants at the top of this header and use them here, something like `EGRESS_GATEWAY_NO_GATEWAY` and `EGRESS_GATEWAY_EXCLUDED_CIDR`